QuickFIXn/ThreadedSocketAcceptor.cs

```diff
@@ -274,6 +274,8 @@ public void Stop(bool force)
             LogoutAllSessions(force);
             DisposeSessions();
             _sessions.Clear();
+            _nonSessionLog.Dispose();
+            _isStarted = false;
```

Comment:
When the `Acceptor` stops, the `_isStarted` field value should be set to `false`